resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientApplication.scala

```diff
@@ -133,6 +134,17 @@ private[spark] class Client(
       .build()
     val driverPodName = resolvedDriverPod.getMetadata.getName
 
+    val preKubernetesResources = resolvedDriverSpec.preKuberenetesResources
+    for (resource <- preKubernetesResources) {
+      resource match {
+        case pg: PodGroup =>
+          val pgClient = V1Beta1.getClient(kubernetesClient, pg.getMetadata.getNamespace)
```

Comment:
It would be replaced by https://github.com/fabric8io/kubernetes-client/pull/3580